### PR TITLE
Creation of Moderation Guidelines Quiz

### PR DESCRIPTION
**Pull request by aFinchy**
Initial creation of Moderation Guidelines Quiz.

### CODE REVIEW

File: process-docs-and-markdown/discord-moderation-guidelines-Quiz.md [hunk 1]
```diff
@@ -0,0 +1,982 @@
...
+Answer: C) To ensure important information and data are not lost in the event of an accident or technical issue.<br> 
+<br> 
+
+### Helpful Resources and Links
+
+N/A <br> 
+
+### Contact Information for Support
+
+N/A <br>
```
Comment (chanmosq) [inline process-docs-and-markdown/discord-moderation-guidelines-Quiz.md:946]: Remove these sections if there is no content 

```suggestion
```

File: process-docs-and-markdown/discord-moderation-guidelines-Quiz.md [hunk 2]
```diff
@@ -0,0 +1,982 @@
...
+Answer: C) To enforce the community's standards and guidelines <br> 
+<br> 
+
+### Commitment to Upholding Guidelines and Policies
+
+N/A <br> 
+
+### Final Remarks
+
+N/A <br>
```
Comment (chanmosq) [inline process-docs-and-markdown/discord-moderation-guidelines-Quiz.md:982]: Remove if there's no contents 
```suggestion
```

File: process-docs-and-markdown/discord-moderation-guidelines-Quiz.md [hunk 3]
```diff
@@ -0,0 +1,982 @@
+# Discord Moderation Guidelines Quiz
+
+- [II. Discord Community Guidelines](#ii-discord-community-guidelines)
```
Comment (chanmosq) [inline process-docs-and-markdown/discord-moderation-guidelines-Quiz.md:3]: Update the numbered sections so it starts at 1

**Review by chanmosq — CHANGES_REQUESTED** [review]

File: process-docs-and-markdown/discord-moderation-guidelines-Quiz.md (hunk 3 above)
Comment (aFinchy) [reply process-docs-and-markdown/discord-moderation-guidelines-Quiz.md:3]: I was trying to ensure it matches the order from the Moderation Guidelines

**Pull request merged**